### ISSUE TITLE
Separated BN parameters 

### ISSUE THREAD

**Issue by milliema** [KikimorMay/MultiFace#3]
Thanks for the work! The idea is simple yet impressive.
In the code the parameters of BN layers are separated from the rest, and set up with different weight decay and momentum. May I ask why do we need the split? What's the benefit of it?

**Comment by KikimorMay** [comment]
Hi,
We follow the setting of TreB1eN/InsightFace_Pytorch. Maybe you can get the necessary information from this [issue](https://github.com/JiahuiYu/slimmable_networks/issues/15) and this [link](https://discuss.pytorch.org/t/weight-decay-in-the-optimizers-is-a-bad-idea-especially-with-batchnorm/16994)

**Comment by milliema** [comment]
Thanks for the quick response, the information is very helpful.
